src/scripts/librarymenu.js

```diff
@@ -147,6 +155,7 @@ define(["dom", "layoutManager", "inputManager", "connectionManager", "events", "
         }
 
         headerAudioPlayerButton.addEventListener("click", showAudioPlayer);
+        headerSyncButton.addEventListener("click", onSyncButtonClicked);
```

Comment:
where's paired `.removeEventListener()`?

Reply:
I have been following the "style" I found. Looks like the callbacks are "removed" in `renderHeader()` by this line:
```
skinHeader.innerHTML = html;
```

It's ugly, but again, I don't see a place where to remove the event listeners. Rewriting how the whole `librarymenu` works seems out of the scope of this PR.

Am I missing some other hidden file? :sweat_smile:

Reply:
Ohm, what a mess... but for a separate PR certainly.